platform/barefoot/sonic-platform-modules-bfn-newport/debian/rules

```diff
@@ -17,14 +18,16 @@ override_dh_auto_build:
 	make -C $(KERNEL_SRC)/build M=$(MODULE_SRC)
 
 override_dh_auto_install:
-	dh_installdirs -p$(PACKAGE_NAME) $(KERNEL_SRC)/$(INSTALL_MOD_DIR)
-	cp $(MODULE_SRC)/*.ko debian/$(PACKAGE_NAME)/$(KERNEL_SRC)/$(INSTALL_MOD_DIR)
-	dh_installdirs -p$(PACKAGE_NAME) usr/local/bin
-	cp -r $(SCRIPT_SRC)/* debian/$(PACKAGE_NAME)/usr/local/bin
-	dh_installdirs -p$(PACKAGE_NAME) etc/network/interfaces.d/
-	cp -r $(CONFIGS_SRC)/network/interfaces.d/* debian/$(PACKAGE_NAME)/etc/network/interfaces.d/
-	dh_installdirs -p$(PACKAGE_NAME) usr/share/sonic/device/x86_64-accton_as9516bf_32d-r0/plugins
-	cp -r $(PLUGINS_DIR)/*  debian/$(PACKAGE_NAME)/usr/share/sonic/device/x86_64-accton_as9516bf_32d-r0/plugins/
+	(for mod in $(MODULE_NAMES); do \
```

Comment:
Can't we install one package and just use post install step to create sym link?

Reply:
The problem is that the deb package is strictly associated with the specific platform and won't be even installed for another one, so there is no chance for such link creation without making some kind of hack in SONiC build system.